install/ansible-tower/install.yml

```diff
@@ -0,0 +1,13 @@
+id: ansible-tower-integration
+name: Ansible Tower integration
+title: Ansible Tower integration
+description: |
+  The Ansible Tower integration will enable you to monitor and analyze your Ansible Tower infrastructure.
+target:
+  type: integration
+  destination: host
+
+install:
+  mode: link
+  destination:
+    url: https://docs.newrelic.com
```

Comment:
We should provide a better documentation source for installing this setup. Perhaps we can use the [ansible docs](https://docs.newrelic.com/docs/infrastructure/install-infrastructure-agent/config-management-tools/configure-infrastructure-agent-using-ansible/). I'd suggest reaching out to someone on the docs team.

Reply:
on hold, in coordination with @akristen on it.

Reply:
Hi @akristen, request you to kindly provide us the docs url, so that we can update this PR with it. 😃